src/Kusto/examples/Get-AzKustoOperationsResultsLocation.md

```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
```

Comment:
Please fill in the examples and make sure to run `build-module.ps1` to regenerate help files